### PR TITLE
Virtualized/steam rich presence

### PR DESCRIPTION
**Pull request by V-rtualized**
- Got initial achievements enabled status after mods loaded
- Attempted to add steam statuses
    - This doesn't work at the moment because it requires a localization file uploaded in the game's web config, if that happens then it could start working
    - Although, it will group players who are in the same lobby on the friends list

### CODE REVIEW

**Review by TGMM — APPROVED** [review]

File: Multiplayer/Lobby.lua [hunk 1]
```diff
@@ -23,16 +25,36 @@ G.MULTIPLAYER_GAME = {
 	ready_blind_text = "Ready",
 }
 
-PREV_ACHIEVEMENT_VALUE = true
+START_NO_ACHIEVEMENT_VALUE = true
+local init_steamodded_ref = initSteamodded
+function initSteamodded()
+	init_steamodded_ref()
```
[the remark this answers is not in the document]
Comment (V-rtualized) [reply Multiplayer/Lobby.lua:31]: Oh yeah, I am dumb, didn't consider that this is the function that would bring our code into existence haha. Will have to change that to a different one

File: Multiplayer/Lobby.lua [hunk 2]
```diff
@@ -23,16 +25,36 @@ G.MULTIPLAYER_GAME = {
...
+		G.STEAM.friends.setRichPresence('steam_display', '#FullStatus')
+		G.STEAM.friends.setRichPresence('status', '#FullStatus')
+		G.STEAM.friends.setRichPresence('text', 'In Multiplayer Lobby')
+		G.STEAM.friends.setRichPresence('steam_player_group', G.LOBBY.code)
+		G.STEAM.friends.setRichPresence('steam_player_group_size', G.LOBBY.guest.username and '2' or '1')
+	else 
+		G.STEAM.friends.setRichPresence('steam_display', '#FullStatus')
+		G.STEAM.friends.setRichPresence('status', '#FullStatus')
+		G.STEAM.friends.setRichPresence('text', 'Using Multiplayer Mod')
+		G.STEAM.friends.setRichPresence('steam_player_group', '')
```
[the remark this answers is not in the document]
Comment (V-rtualized) [reply Multiplayer/Lobby.lua:56]: This is the full code:

```
if G.LOBBY.code then
		G.STEAM.friends.setRichPresence('steam_display', '#FullStatus')
		G.STEAM.friends.setRichPresence('status', '#FullStatus')
		G.STEAM.friends.setRichPresence('text', 'In Multiplayer Lobby')
		G.STEAM.friends.setRichPresence('steam_player_group', G.LOBBY.code)
		G.STEAM.friends.setRichPresence('steam_player_group_size', G.LOBBY.guest.username and '2' or '1')
	else 
		G.STEAM.friends.setRichPresence('steam_display', '#FullStatus')
		G.STEAM.friends.setRichPresence('status', '#FullStatus')
		G.STEAM.friends.setRichPresence('text', 'Using Multiplayer Mod')
		G.STEAM.friends.setRichPresence('steam_player_group', '')
```

The line you referenced is resetting the group when the player leaves the lobby, but it does equal the lobby code when they are in it

File: Multiplayer/Lobby.lua (hunk 1 above)
[the remark this answers is not in the document]
Comment (V-rtualized) [reply Multiplayer/Lobby.lua]: Resolved this in [1c6cc89](https://github.com/V-rtualized/balatro-multiplayer/pull/18/commits/1c6cc8952139ba24b2226ab82f173ab5876b8deb)

File: Multiplayer/Lobby.lua (hunk 2 above)
[the remark this answers is not in the document]
Comment (TGMM) [reply Multiplayer/Lobby.lua:56]: Oh, sorry about that. I didn't see that part. Thanks for explaining.

**Pull request merged**